### PR TITLE
Add TestStand .seq for NI-FGEN example

### PR DESCRIPTION
**Pull request by synesthetize**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-labview/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Using the DCPower example as a starting point, adds VIs from `TestStand Code Modules` with some modifications
- DCPower specific strings and VIs are converted to use Fgen
- Inline the implementation for anything from `Measurement Service Helpers` since we wish to get away from using that library

Screenshots of new files:
[fgen example - teststand seq.zip](https://github.com/ni/measurementlink-labview/files/12920082/fgen.example.-.teststand.seq.zip)

seq file

### Why should this Pull Request be merged?

[AB#2554497](https://ni.visualstudio.com/DevCentral/_workitems/edit/2554497): Create FGEN example TestStand .seq

### What testing has been done?

Able to run the sequence and observe that the generated report has the expected values.

### CODE REVIEW

File: Source/Example Measurements/NI-FGEN Standard Function/NIFgenStandardFunction.seq
Comment (dixonjoel) [inline Source/Example Measurements/NI-FGEN Standard Function/NIFgenStandardFunction.seq:1]: image

This needs to be saved in a TS no later than 2021 SP1.

**Review by dixonjoel — CHANGES_REQUESTED** [review]

File: Source/Example Measurements/NI-FGEN Standard Function/NIFgenStandardFunction.seq
Comment (synesthetize) [reply Source/Example Measurements/NI-FGEN Standard Function/NIFgenStandardFunction.seq:1]: Good catch, didn't realize that would be an issue.

Comment (synesthetize) [reply Source/Example Measurements/NI-FGEN Standard Function/NIFgenStandardFunction.seq:1]: Fixed in the latest commit. Tested this in TS 2021 and confirmed the sequence opens and works.

**Review by DelpireNI — APPROVED** [review]
Copied your VI's and updated them for the scope example and they worked great!

**Review by jasonmreding — APPROVED** [review]

**Review by dixonjoel — APPROVED** [review]

**Pull request merged**